pages/doc/integrations_tas_alerts.md

```diff
@@ -468,65 +449,72 @@ Measures the total PPS received by the Wavefront proxy.
 
 ## TAS MySQL Status
 
-The TAS backing database is used for persistent data storage by several TAS components. This alert uses the metric from
-the SQL database used by system components, not from the MySQL service used by applications. TAS components that use
-system databases include the Cloud Controller, Diego Brain, Gorouter, and User Authorization and Authentication (UAA)
+The TAS backing database is used for persistent data storage by several TAS components. This alert uses the metric from the SQL database used by system components, not from the MySQL service used by applications. TAS components that use system databases include the Cloud Controller, Diego Brain, Gorouter, and User Authorization and Authentication (UAA)
 server. See [TAS Components](https://docs.pivotal.io/application-service/concepts/architecture/index.html) for details.
 
-`1` indicates that TAS backing database is available, and `0` indicates that it is unavailable.
+* 1 indicates that TAS backing database is available
+* 0 indicates that TAS backing database is unavailable
 
 **Troubleshooting**
 
-1. Inspect the troubleshooting metrics on the TAS: Platform MySQL dashboard.
+1. Inspect the troubleshooting metrics on the **Platform MySQL** dashboard of the TAS integration.
```

Comment:
The full dashboard name here is "TAS: Platform MySQL". I am inclined to keep it, because there are several MySQL dashboards (including a "TAS Services: MySQL" also published by us) and we want to be unambiguous.

Reply:
Sure, I updated all dashboard names to include "TAS:".